tests/core/test_taipy.py

```diff
@@ -439,36 +431,48 @@ def test_compare_scenarios(self, scenario):
             mck.assert_called_once_with(scenario, scenario, data_node_config_id="dn")
 
     def test_subscribe_scenario(self, scenario):
+        def cb(s, j):
+            print()
```

Comment:
Should we put this function at the file scope?
So that we don't have to redefine it multiple times